### ISSUE TITLE
Missing keyboard support

### ISSUE THREAD

**Issue by witek** [slgobinath/SafeEyes#103]
Please add keyboard support to the break screen. It would be nice to skip by pressing `ESC` or `s`.

**Comment by Joe5** [comment]
I also don't like it. I get the idea, but I don't think that many people need app to catch keystrokes so they don't do something on the background. I would suggest to release the keyboard after 5 seconds, it should be more than enough not to run something on the background when the overlay is displayed.

My biggest problem is that I am unable to use my media keyboard keys to pause my music playback.

**Comment by slgobinath** [comment]
@witek, @Joe5,
I agree with your points. In the case of @witek's requirement, we must be able to skip the break using the keyboard shortcut. Based on @Joe5's requirement you must be able to control the audio/video player using the keyboard. Both make sense and I also have the same requirements.

The main purpose of blocking keyboard is avoiding shortcuts to bypass the break screen. For example `Alt + F4` or at least `Alt + Tab` is enough to bypass the break screen. If that happens, the whole point of break screen is useless (We use Safe Eyes to force us to take a break). What I am thinking about is allowing `Esc` to skip break (not in strict break mode) and letting the user use multimedia keys.

What do you think?

**Comment by Joe5** [comment]
Good point with `Alt + F4 / Alt + Tab` I didn't considered that. Anyway I think it should be locked for ~5 seconds even on escape to avoid unintentional break skip.

**Comment by witek** [comment]
I think, **every** action that is possible with the mouse, should be possible with the keyboard too.

Sometimes I disconnect my mouse to practice keyboard usage, which makes me faster in the long run. Having applications which do not work then is just sad.

Please allow a key for every action. Like `ctrl+s` for skip and `ctrl+p` for postpone.

**Comment by slgobinath** [comment]
Added with https://github.com/slgobinath/SafeEyes/commit/4509aa1d189343dc51332c209714ba861c594fc2 and https://github.com/slgobinath/SafeEyes/commit/ca65add8e836decaf883623539695d235c727ca3. Default keyboard shortcuts are:
 - `Esc` - skip
 - `Space` - postpone.

Users can change the keyboard to any other single key presses (For the moment, the combination of keys are not supported). The keyboard shortcut will work only if those buttons are visible.

@Joe5 thanks for your suggestion. The keyboard shortcut will be enabled only after 2 seconds to avoid unintentional key presses. The 2 sec delay can be configured by the user through the settings dialog.

**Comment by bitterjug** [comment]
Is there a way to trigger take a break now from the keyboard?
